src/cloud-cli/login.ts

```diff
@@ -74,7 +74,8 @@ export async function login(username: string): Promise<number> {
     const response = await axios.request(deviceCodeRequest);
     deviceCodeResponse = response.data as DeviceCodeResponse;
   } catch (e) {
-    logger.error(new Error(`failed to log in: ${(e as Error).message}`));
+    (e as Error).message = `failed to log in: ${(e as Error).message}`;
```

Comment:
What does this do in practice?

Reply:
Yes we just improves the message

Reply:
Compared to the original code (not the first PR), it seems it improves the message vs allowing the logger to make the same improvement?

Reply:
What would `allow the logger to make the same improvement` mean here? (The idea is to keep the logger interface simple and as close as Winston as possible, i.e., for errors, you can pass an error object, a string message or a `JSON.stringify`-able object)